PanModal/Presentable/PanModalPresentable.swift

```diff
@@ -11,227 +11,240 @@ import UIKit
 /**
  This is the configuration object for a view controller
  that will be presented using the PanModal transition.
-
+ 
  Usage:
  ```
  extension YourViewController: PanModalPresentable {
-    func shouldRoundTopCorners: Bool { return false }
+ func shouldRoundTopCorners: Bool { return false }
  }
  ```
  */
 public protocol PanModalPresentable: AnyObject {
-
+    
     /**
      The scroll view embedded in the view controller.
      Setting this value allows for seamless transition scrolling between the embedded scroll view
      and the pan modal container view.
      */
     var panScrollable: UIScrollView? { get }
-
+    
     /**
      The offset between the top of the screen and the top of the pan modal container view.
-
+     
      Default value is the topLayoutGuide.length + 21.0.
      */
     var topOffset: CGFloat { get }
-
+    
     /**
      The height of the pan modal container view
      when in the shortForm presentation state.
-
+     
      This value is capped to .max, if provided value exceeds the space available.
-
+     
      Default value is the longFormHeight.
      */
     var shortFormHeight: PanModalHeight { get }
-
+    
     /**
      The height of the pan modal container view
      when in the longForm presentation state.
      
      This value is capped to .max, if provided value exceeds the space available.
-
+     
      Default value is .max.
      */
     var longFormHeight: PanModalHeight { get }
-
+    
```

Comment:
Looks like your IDE filled in whitespaces for no reason here. Double check if you Xcode trims these.
Xcode -> Preferences -> Text Editing -> Editing -> While Editing -> Check`Automatically trim trailing whitespace` and `Including whitespace-only lines`